Cesium.Parser/CParser.cs

```diff
@@ -123,6 +123,10 @@ public partial class CParser
     private static Expression MakePrefixIncrementExpression(ICToken _, Expression target) =>
         new PrefixIncrementExpression(target);
 
+    [Rule("unary_expression: '~' unary_expression")]
+    private static Expression MakeBitwiseNotExpression(ICToken _, Expression target) =>
+        new BitwiseNotExpression(target);
+
```

Comment:
I'd prefer this to be closer to the AST provided by the standard. Something like this:
```suggestion
    [Rule("unary_expression: unary_operator cast_expression")]
    private static Expression MakeBitwiseNotExpression(ICToken @operator, Expression target) =>
        new UnaryExpression(@operator, target);

    // TODO: unary-operator: one of
    //    & * + - !
    [Rule("unary_operator: '~'")]
    // TODO: [Rule("unary_operator: '!'")]
    private static Expression MakeUnaryOperator(ICToken @operator) => @operator;
```

Also note how the AST from the standard dictates changes in the AST records: instead of `Ast.BitwiseNotExpression`, I'd prefer to see `Ast.UnaryExpression`.

You're free to leave `Ir.Expressions.BitwiseNotExpression` as-is, though. That's okay.